### PR TITLE
Fix ignore of unexpected files in key dir

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**

### CODE REVIEW

**Comment by fjl** [comment]
Please don't do *new(sliceType). Use make(sliceType) instead. The last two arguments to make are optional.

File: crypto/key_store_plain.go [hunk 1]
```diff
@@ -117,13 +117,13 @@ func GetKeyAddresses(keysDirPath string) (addresses [][]byte, err error) {
 	if err != nil {
 		return nil, err
 	}
-	addresses = make([][]byte, len(fileInfos))
-	for i, fileInfo := range fileInfos {
+	addresses = *new([][]byte)
```
Comment (obscuren) [inline crypto/key_store_plain.go]: Please explain this ..

**Comment by Gustav-Simonsson** [comment]
@fjl Actually only the third is optional: https://golang.org/ref/spec#Making_slices_maps_and_channels

**Comment by fjl** [comment]
Whoa. You caught me there. Anyway, looks better now.

**Comment by obscuren** [comment]
`var addresses [][]byte`?

**Comment by Gustav-Simonsson** [comment]
make -> new -> make -> variable declaration -> function signature parameter name

Anything else we can optimize while we're at it? :)

**Pull request merged**